src/task.c

```diff
@@ -344,8 +344,13 @@ static void UNUSED sub_80028DC(void)
                 cur->state += ((struct IwramNode *)IWRAM_PTR(cur->next))->state;
                 cur->next = ((struct IwramNode *)IWRAM_PTR(cur->next))->next;
             } else {
-                nextNodeSpace = (u32)cur->next
-                    + (u8 *)IWRAM_PTR(offsetof(struct IwramNode, space));
+#if PLATFORM_GBA
+                nextNodeSpace
+                    = (void *)(cur->next + IWRAM_PTR(offsetof(struct IwramNode, space)));
+#else
+                nextNodeSpace
+                    = (void *)cur->next + IWRAM_PTR(offsetof(struct IwramNode, space));
```

Comment:
I'm sure this wouldn't compile with MSVC since it doesn't like arithmetic with void pointers, but I'd have to double check. It's not like we'll ever use it, anyway.

Reply:
This isn't the only place doing that so if we do need to fix we can do across the whole codebase at the same time